nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -56,12 +61,135 @@ void builder2file(const flatbuffers::FlatBufferBuilder &builder,
  *
  */
 class TfOpNode {
+public:
+  using Variables = std::vector<const Var_Grad *>;
+
+  TfOpNode() = default;
+
+  /**
+   * @brief Construct a new Tf Op Node object from layer
+   * @note this is a shortcut to skip if layer does not need to be devided or
+   * fused
+   * @param layer layer that is converted to TfOpNode
+   */
+  TfOpNode(const Layer &layer) {
+    setInputs(layer.getInputRef());
```

Comment:
IMO for cleaner code, setInputs, setOutputs, setWeights can use a common implementation or the same static function with parameters to set.

Reply:
Could you give me an example? You mean templated code?

Reply:
Yes, a templated common function which these functions can call. Feel free to do in a later PR.

Reply:
Okay